### PR TITLE
#19 Split model into several parts

### PR DESCRIPTION
**Pull request by zelenyhleb**
Added markup bundle pack

Signed-off-by: Nikifor Fedorov nikifor.fedorov@arsysop.ru

### CODE REVIEW

**Comment by zelenyhleb** [comment]
**Merge conflict note (for a reviewer):** after merging #39 there will be a merge conflict in aggregator's pom. To resolve it, just place all the added references next to each other in the file, no need to work with this conflict additionally somehow.

File: bundles/ru.arsysop.loft.rgm.markup.edit/.classpath [hunk 1]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src-gen"/>
+	<classpathentry kind="src" path="src"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11"/>
```
Comment (ruspl-afed) [inline bundles/ru.arsysop.loft.rgm.markup.edit/.classpath:5]: 1.8

File: bundles/ru.arsysop.loft.rgm.markup.model/.classpath [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
+	<classpathentry kind="src" path="src-gen"/>
+	<classpathentry kind="con" path="org.eclipse.jdt.launching.JRE_CONTAINER/org.eclipse.jdt.internal.debug.ui.launcher.StandardVMType/JavaSE-11"/>
```
Comment (ruspl-afed) [inline bundles/ru.arsysop.loft.rgm.markup.model/.classpath:4]: 1.8

**Review by ruspl-afed — APPROVED** [review]

**Review by ruspl-afed — APPROVED** [review]
Thanks @zelenyhleb

**Pull request merged**